jena-arq/Grammar/master.jj

```diff
@@ -2045,6 +2045,9 @@ Expr Aggregate() : { Aggregator agg = null ; String sep = null ;
   | t = <AVG> <LPAREN> ( <DISTINCT> { distinct = true ; } )? expr = Expression() <RPAREN>
     { agg = AggregatorFactory.createAvg(distinct, expr) ; }
 
+  | t = <MEDIAN> <LPAREN> ( <DISTINCT> { distinct = true ; } )? expr = Expression() <RPAREN>
```

Comment:
This should be fenced with an `#ifdef` so it is only incorporated into the ARQ syntax